pokedex-with-bit/README.md

```diff
@@ -57,117 +39,63 @@ docker run -p 45678:45678 jinahub/app.example.pokedexwithbit:0.0.1-0.9.20
-<td> index.yml</td>
-<td> <a href="https://github.com/jina-ai/dashboard">Flow in Dashboard</a></td>
-</tr>
-<tr>
-<td>
-
-```python
-    f = Flow.load_config('flows/index.yml')
+#### Diving Deeper
 
-    with f:
-        f.index_files(image_src, request_size=64, read_mode='rb', size=num_docs)
-```
+Jina's REST API uses the [data URI scheme](https://en.wikipedia.org/wiki/Data_URI_scheme) to represent multimedia data. To query your indexed data, simply organize your picture(s) into this scheme and send a POST request to `http://0.0.0.0:45678/api/search`, e.g.:
 
-</td>
-<td>
-  <sub>
-
-```yaml
-!Flow
-version: '1'
-pods:
-  - name: crafter
-    show_exc_info: true
-    uses: pods/craft.yml
-    shards: $JINA_SHARDS
-    read_only: true
-  - name: encoder
-    show_exc_info: true
-    uses: pods/encode.yml
-    shards: $JINA_SHARDS
-    timeout_ready: 600000
-    read_only: true
-  - name: vec_idx
-    show_exc_info: true
-    uses: pods/vec.yml
-    shards: $JINA_SHARDS_INDEXERS
-    polling: any
-    timeout_ready: 100000 # larger timeout as in query time will read all the data
-  - name: doc_idx
-    uses: pods/doc.yml
-    shards: $JINA_SHARDS_INDEXERS
-    polling: any
-    timeout_ready: 100000 # larger timeout as in query time will read all the data
-    needs: [gateway]
-  - name: join_all
-    uses: _merge
-    needs: [doc_idx, vec_idx]
-    read_only: true
+```bash
+curl --verbose --request POST -d '{"top_k": 10, "mode": "search",  "data": ["data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAICAIAAABLbSncAAAA2ElEQVR4nADIADf/AxWcWRUeCEeBO68T3u1qLWarHqMaxDnxhAEaLh0Ssu6ZGfnKcjP4CeDLoJok3o4aOPYAJocsjktZfo4Z7Q/WR1UTgppAAdguAhR+AUm9AnqRH2jgdBZ0R+kKxAFoAME32BL7fwQbcLzhw+dXMmY9BS9K8EarXyWLH8VYK1MACkxlLTY4Eh69XfjpROqjE7P0AeBx6DGmA8/lRRlTCmPkL196pC0aWBkVs2wyjqb/LABVYL8Xgeomjl3VtEMxAeaUrGvnIawVh/oBAAD///GwU6v3yCoVAAAAAElFTkSuQmCC", "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAICAIAAABLbSncAAAA2ElEQVR4nADIADf/AvdGjTZeOlQq07xSYPgJjlWRwfWEBx2+CgAVrPrP+O5ghhOa+a0cocoWnaMJFAsBuCQCgiJOKDBcIQTiLieOrPD/cp/6iZ/Iu4HqAh5dGzggIQVJI3WqTxwVTDjs5XJOy38AlgHoaKgY+xJEXeFTyR7FOfF7JNWjs3b8evQE6B2dTDvQZx3n3Rz6rgOtVlaZRLvR9geCAxuY3G+0mepEAhrTISES3bwPWYYi48OUrQOc//IaJeij9xZGGmDIG9kc73fNI7eA8VMBAAD//0SxXMMT90UdAAAAAElFTkSuQmCC"]}' -H 'Content-Type: application/json' 'http://0.0.0.0:34567/api/search'
 ```
 
-</sub>
+[JSON payload syntax and spec can be found in the docs](https://docs.jina.ai/chapters/restapi/#).
 
-</td>
-<td>
+The above explains how to use a REST gateway, but by default Jina uses a gRPC gateway, which has much higher performance and richer features. Read our [documentation on Jina IO](https://docs.jina.ai/chapters/io/#) for more information.
 
-![Flow in Dashboard](.github/.README_images/6d28795b.png?raw=true)
+### Build a Docker Image
 
-</td>
-</tr>
-</table>
+After playing with it for a while, you may want to change the code and rebuild the image. Simply run:
+```bash
+docker build -t jinaai/app.examples.pokedexwithbit .
```

Comment:
I think this is wrong, u need the -f Dockerfile?

Reply:
I only polished it up, didn't check all commands. But I've run commands exactly the same as that (except image name) before and they've worked fine. As long as there's a `Dockerfile` in the dir, it'll automatically use it so `-f` is optional

Reply:
okk